bin/compile

```diff
@@ -13,6 +13,14 @@ unset GIT_DIR
 BUILD_DIR=$1
 BUILDPACK_DIR="$(dirname $(dirname $0))"
 
+STUNNEL=$(command -v stunnel4)
+
+if [ -z $STUNNEL ]; then
+    echo "------> stunnel not detected! stunnel not supported on heroku-24+"
+    echo "------> this buildpack is not required for Redis 6+: https://devcenter.heroku.com/articles/heroku-redis#security-and-compliance"
```

Comment:
For buildpack error messages, the typical styling is to use an exclamation mark prefix (` !     `) instead of the `----->`.

Also , what do you think about including instructions on how to remove the buildpack?

For example something like:

```
 !     stunnel not detected! stunnel not supported on heroku-24+
 !     This buildpack is not required for Redis 6+. Remove this buildpack using:
 !         $ heroku buildpacks:remove heroku/redis
 !     And then follow the instructions for using Redis' native TLS support:
 !     https://devcenter.heroku.com/articles/heroku-redis#security-and-compliance
```

Reply:
That works for me - it will need to go through CX fore view.